iphone/Classes/TiUIiOSStepper.m

```diff
@@ -0,0 +1,204 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSSTEPPER
+#import "TiUIiOSStepper.h"
+#import "TiUIiOSStepperProxy.h"
+#import "TiUtils.h"
+
+@implementation TiUIiOSStepper
+
+-(UIStepper*)stepper
+{
+    if (stepper ==nil)
+    {
+        stepper = [[[UIStepper alloc]init] retain];
+        [stepper setMaximumValue:100];
```

Comment:
Ive added this so if the user doesn't set a value this will be the default one. If a user sets a value it will be set and the value set here will be overwritten.